INSTALL

```diff
@@ -31,3 +31,18 @@ If you want to supply an install prefix to cmake, add the flag:
 
 Where <prefix> can be /usr, /usr/local, /opt, etc. The default is /usr/local.
 
+
+
+Building with QT5
+
+First of all please note that CMake >= 2.8.11 is required for building with
+Qt5 support. In order to build LMMS with Qt5, add the following flag when
+invoking cmake:
+
+-DWANT_QT5=ON
+
+If your Qt5 installation does not reside in standard installation paths,
+additionally pass e.g.
+
+-DCMAKE_PREFIX_PATH=/opt/qt53/
+
```

Comment:
As the Qt5 build instructions are removed from the INSTALL.Qt5 I think they should be added on the "Compiling lmms" wiki page so this information is not lost for anyone who would work on the Qt5 port (perhaps me :)
Any comments?